src/utils/PageEmbed.ts

```diff
@@ -313,8 +317,6 @@ export class PageEmbed extends EmitterBase
 
                 if(raw.match(/[\d]+/) && !isNaN(num))
                 {
-                    autoDel(msg);
```

Comment:
why?

Reply:
nvm

Reply:
that's what's causing the double delete. its first deleted after the 2 if statements then the timeout happens and it tries to delete it again

Reply:
yea I just noticed lol